pages/resources/quests.mdx

```diff
@@ -202,18 +211,18 @@ Represents the user's quest progression.
-| claimed_at                | ISO8601 timestamp | When the user claimed the quest's reward                                                                                        |
-| claimed_tier              | unknown           | Unused                                                                                                                          |
-| last_stream_heartbeat_ack | ISO8601 timestamp | Timestamp of the latest stream heartbeat request                                                                                |
-| stream_progress_seconds   | ISO8601 timestamp | The total amount of seconds the user has been streaming the game for since the quest started and before the quest was completed |
-| dismissed_quest_content   | ISO8601 timestamp | Whenever the user has dismissed the quest                                                                                       |
-| progress                  | object            | Unused object that could have been an early iteration of the user status object                                                 |
+| Field                     | Type               | Description                                                                                                                                                         |
+| ------------------------- | ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| user_id                   | snowflake          | The user's ID                                                                                                                                                       |
+| quest_id                  | snowflake          | The ID of the quest                                                                                                                                                 |
+| enrolled_at               | ISO8601 timestamp  | When the user accepted the quest                                                                                                                                    |
+| completed_at              | ?ISO8601 timestamp | When the user completed the quest                                                                                                                                   |
+| claimed_at                | ?ISO8601 timestamp | When the user claimed the quest's reward                                                                                                                            |
+| claimed_tier              | ?integer           | Which reward tier the user has claimed, if the quest's [`assignment_method`](#quest-rewards-config-structure) is set to [`TIERED`](#quest-reward-assignment-method) |
```

Comment:
Since this was changed here, shouldn't it also be changed under Quest Reward Code Structure?